docs/queries/disjunction_max.rst

```diff
@@ -0,0 +1,13 @@
+Disjunction Max Query
+==========================
+
+A query that generates the union of documents produced by its subqueries, and that scores each document with the maximum score for that document as produced by any subquery, plus a tie breaking increment for any additional matching subqueries. This is useful when searching for a word in multiple fields with different boost factors (so that the fields cannot be combined equivalently into a single search field). This maps to DisjunctionMaxQuery in Lucene.
```

Comment:
I think creating an example of putting data in Nrtserach and then searching them using this query would be easier to understand than just the pure description. This can be done for the queries that are as straightforward as some others such as `exists`. We can do that in a separate PR.

Reply:
Yes we should do that, I'm only making it easier to understand query syntax with this PR.